### ISSUE TITLE
The datasets of gen_data.py

### ISSUE THREAD

**Issue by zhouwy-lv** [pxiangwu/MotionNet#14]
Hello！
The trainval on the nuscenes official website is divided into 10 parts. Do I need to download all the datasets of these parts?

**Comment by pxiangwu** [comment]
Hello,

Yes, you need to download all the 10 parts to obtain a complete dataset. In total there are 850 scenes.
<img width="1142" alt="image" src="https://user-images.githubusercontent.com/3341256/163667503-86b927eb-0965-4321-81c0-8d2fd3ec596a.png">

**Comment by zhouwy-lv** [comment]
Thanks for your replay! In addition, the v1.0_mini dataset seems to have problems when running train_single_seq.
First of all, shuffle must be false to be executed.
trainloader = torch.utils.data.DataLoader(trainset, batch_size=BATCH_SIZE, shuffle=True, num_workers=num_workers)
But, The training datasize is 0.
While in train_muti_seq, shuffle can be true, and the datasize is 138.
